### DISCUSSION TITLE
Alert entity:  change to allow template conditions, snoozing, events, ...

### DISCUSSION THREAD

**Discussion by redstone99** [home-assistant/architecture#1013] [Entity Models]
Hi, I had some ideas for improving the [Alert](https://www.home-assistant.io/integrations/alert/) entity and was wondering what people thought. Because it involves substantial changes, I coded it up as a separate component and put it in github: [Alert2](https://github.com/redstone99/hass-alert2).  The main features are:
- **Native event-based alerting**. No need to approximate it with conditions and time windows.
- **Template conditions**.  No need for extra binary sensors. Also means the logic for an alert is in one place in your config file, which makes it easier to manage.
- **Snooze / disable notifications**. Handy for alerts that fire frequently. Example: a temperature sensor that hovers right around a threshold.
- **Persistent notification details**. In your HA dashboard, you can view past alert firings as well as the message text sent in notifications.
- **Custom front-end card**. Makes it easier to view and manage recent alerts.

The front-end Lovelace card shows recently active alerts, with controls for ack’ing:

![Alert2 overview lovelace card](https://raw.githubusercontent.com/redstone99/hass-alert2/master/resources/overview.png)

And here’s what the more-info dialog looks like when clicking on an alert2 entity. It shows firing history as well as notification messages sent, and controls for snoozing and disabling:

![Alert2 more-info dialog](https://raw.githubusercontent.com/redstone99/hass-alert2/master/resources/more-info.png)

-Josh

**Comment by codeisnice** [comment]
Hi, I use a helper (input_number) for logging the outside temperature. Since the last update, this sensor appears in a separate chart in the history. Why?

**Reply by parautenbach** [reply]
What does this have to do with this issue?

You should post support questions on the forum. 

I think there was a change to not only group history charts on the history dashboard by UoM, but rather UoM _and_ domain. I can't find the information now, so I stand corrected. 

Please check and ask on the forum.
